lib/chart/internal/guards.ex

```diff
@@ -2,7 +2,13 @@ defmodule Chart.Internal.Guards do
   @moduledoc """
   A gurad helpers
   """
+
+  defguard is_decimals(d) when is_integer(d) and 0 <= d
+  defguard is_nonnegative_number(x) when is_number(x) and 0 <= x
+  defguard is_nonnegative_numbers(x, y) when is_number(x) and is_number(y) and 0 <= x and 0 <= y
```

Comment:
proc?

Reply:
mi to tu zustalo zda se jako relikt, ponechal bych..